<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="subscribetopic">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Server Adress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serveradress"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Client ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="clientid"/>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Number of messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numofmsg"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Topic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="topic"/>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Subsribe</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="msgtree">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>Topic</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dashboard"/>
      <widget class="QWidget" name="subscribepublish">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>23</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CURRENT TOPIC</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>New topic to add (sub topic of this one):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TOPIC:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Message to publish:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="currenttopic">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Subsribe topic</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Publish to topic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionaddSubscribeTopic"/>
    <addaction name="actionshowTopicMsg"/>
    <addaction name="actionDashBoard"/>
    <addaction name="actionSave_current_State"/>
    <addaction name="actionSubscribe_publish_current"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionaddSubscribeTopic">
   <property name="text">
    <string>add new Subscribe Topic</string>
   </property>
  </action>
  <action name="actionshowTopicMsg">
   <property name="text">
    <string>Topics Messages show</string>
   </property>
  </action>
  <action name="actionDashBoard">
   <property name="text">
    <string>DashBoard</string>
   </property>
  </action>
  <action name="actionaddedtolist">
   <property name="text">
    <string>addedtolist</string>
   </property>
  </action>
  <action name="actionSave_current_State">
   <property name="text">
    <string>Save current State</string>
   </property>
  </action>
  <action name="actionSubscribe_publish_current">
   <property name="text">
    <string>Subscribe/publish current</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
